app/model/Email.scala

```diff
@@ -157,7 +157,7 @@ object PaperFieldsGenerator {
                  promotionDescription: Option[String]
                ): Seq[(String, String)] = {
     val paymentFields = paymentMethod match {
-      case GoCardless(mandateId, accName, accNumber, sortCode) => Seq(
+      case GoCardless(mandateId, accName, accNumber, sortCode, _, _) => Seq(
```

Comment:
What are the underscores for? Are they wildcard arguments?

Reply:
It's a match pattern so we're saying that we want to match a case where it is of type `GoCardless` and we want to capture the the first four fields of that object to named variables which we can use below, but we don't care about the last two fields because we're not going to use them.

Reply:
Ah, thanks!